android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -1888,9 +1888,12 @@ public static boolean updateText(KeyboardType kbType, String text) {
 
   public static boolean updateSelectionRange(KeyboardType kbType, int selStart, int selEnd) {
     boolean result = false;
+    // Currently, KMW doesn't support selection on touch alias elements
```

Comment:
Touch alias elements are not relevant to Keyman for Android -- they are only used in KeymanWeb in a browser.